scripts/monitoring/auto_scale_eks_nodes.py

```diff
@@ -0,0 +1,267 @@
+import os
+import time
+import warnings
+from datetime import datetime
+
+import boto3
+import pytz
+import requests
+from dateutil.parser import parse
+from prometheus_api_client import PrometheusConnect
+
+from scripts.workers.worker_utils import EvalAI_Interface
+
+warnings.filterwarnings("ignore")
+
+utc = pytz.UTC
+NUM_PROCESSED_SUBMISSIONS = "num_processed_submissions"
+NUM_SUBMISSIONS_IN_QUEUE = "num_submissions_in_queue"
+PROMETHEUS_URL = os.environ.get(
+    "MONITORING_API_URL", "https://monitoring-staging.eval.ai/prometheus/"
+)
+
+PROD_INCLUDED_CHALLENGE_QUEUES = [
+    "habitat-rearrangement-challenge-2022-1820-production-e900ef02-eeb1-439a-b4a4-833",
+    "habitat-challenge-2022-1615-production-be08fce5-72a7-40bc-aa2e-4df7b3380a8a",
```

Comment:
Can we also add habitat challenge 2021 to this list?

Reply:
@Ram81
I am adding separate lists for challenges on both prod and staging to handle the `ENV=dev` case.